examples/seismic/tutorials/14_creating_synthetics.ipynb

```diff
@@ -48,7 +48,8 @@
      "name": "stdout",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.    try:
</pre><p>Should we have version check for all python packages?</p><p>Maybe you could also try to see whether newer pyvista or gempy versions can be available?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1895/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>